spec/my_enumerables_spec.rb

```diff
@@ -181,15 +184,13 @@
     end
   end
 
-  let(:test_range) { (1..4) }
-
   describe '#my_map' do
     it 'returns a modified array resulting from block operation' do
-      expect(test_arr.my_map { |i| i * i }).to eq([1, 4, 9])
+      expect(test_arr.my_map(&test_block)).to eq([2, 4, 6])
```

Comment:
Not crazy about the `eq` having hardcoded values. You'll get better at this with practice

Reply:
Oh darn, missed that one, thanks for catching that!